ideascube/search/migrations/0002_reindex.py

```diff
@@ -1,20 +0,0 @@
-# -*- coding: utf-8 -*-
-from __future__ import unicode_literals
-
-from django.db import migrations, connections
-from ideascube.search.utils import reindex_content
-
-
-def update_reindex(*args):
-    cursor_default = connections['default'].cursor()
-    cursor_default.execute("DROP TABLE IF EXISTS idx")
```

Comment:
Yes, that better than simply removing it.

(keeping the "IF EXISTS" in the request as 0001_initial do not create the db anymore)
Thinking of that, does 0001_initial have to create the index table in transient as it was creating the table in default before ? I don't think it is a problem, but I prefer ask now than fixing a bug later.